test_app/settings.py

```diff
@@ -23,7 +23,7 @@
 
```

Comment:
Hmm. I see. My original thought was that when using the `admin_approval` backend it should be implied that you have set your `ADMINS`. How do you think I should ensure that?